Error on API doc page with Alma instance

See https://alma.lobid.org/resources/api

From the logs (thanks for reporting @TobiasNx):

```
Caused by: java.lang.NullPointerException: null
        at views.html.api_Scope0$api.sample_doc$1(api.template.scala:40)
        at views.html.api_Scope0$api.apply(api.template.scala:156)
```

--- a/web/app/views/api.scala.html
+++ b/web/app/views/api.scala.html
@@ -41,7 +41,7 @@ $('input.search-resources').autocomplete({
 	<h2 id='basics'>API-Grundlagen <small><a href='#basics'><span class='glyphicon glyphicon-link'></span></a></small></h2>
 
 	<dl>
-	@desc("HTTP GET", resources.routes.Application.resource("HT018472857", format="json"))
+	@desc("HTTP GET", resources.routes.Application.resource("990150876610206441", format="json"))
 	@desc("Suche über alle Felder", resources.routes.Application.query("kunst", format="json"))
 	@desc("Suche mit Paginierung", resources.routes.Application.query("kunst", from = 2, size = 2, format="json"))
 	@desc("Einschränkung auf Bestand in Bibliotheken", resources.routes.Application.query("kunst", owner = "DE-290,DE-294", format="json"))
@@ -66,16 +66,16 @@ $('input.search-resources').autocomplete({
 	<h2 id='content_types'>Inhaltstypen <small><a href='#content_types'><span class='glyphicon glyphicon-link'></span></a></small></h2>
 
 	<p>Standardmäßig liefert dieser Dienst strukturierte API-Antworten (als JSON):</p>
-	<p><code>curl https://lobid.org@resources.routes.Application.resource("HT018472857")</code></p>
+	<p><code>curl https://lobid.org@resources.routes.Application.resource("990150876610206441")</code></p>
 	<p>Er unterstützt Content-Negotiation für JSON (application/json), RSS (application/rss+xml) und HTML (text/html):</p>
 	<p><code>curl --header "Accept: application/json" https://lobid.org@resources.routes.Application.query("kunst")</code></p>
 	<p><code>curl --header "Accept: application/rss+xml" https://lobid.org@resources.routes.Application.query("kunst")</code></p>
 	<p>Der Query-Parameter "format" (Werte: html,json,rss) kann verwendet werden, um den Accept-Header aufzuheben, z.B. zur Anzeige von JSON im Browser:</p>
-	<p><a href='@resources.routes.Application.resource("HT018472857", format="json")'>@resources.routes.Application.resource("HT018472857", format="json")</a></p>
+	<p><a href='@resources.routes.Application.resource("990150876610206441", format="json")'>@resources.routes.Application.resource("990150876610206441", format="json")</a></p>
 	<p>Oder zum Abonnieren eines RSS-Feeds:</p>
 	<p><a href='@resources.routes.Application.query("kunst", format="rss")'>@resources.routes.Application.query("kunst", format="rss")</a></p>
 	<p>Der Wert des Format-Parameters kann für Einzeltreffer auch in URLs als Dateiendung verwendet werden:</p>
-	<p><a href='@resources.routes.Application.resourceDotFormat("HT018472857", format="json")'>@resources.routes.Application.resourceDotFormat("HT018472857", format="json")</a></p>
+	<p><a href='@resources.routes.Application.resourceDotFormat("990150876610206441", format="json")'>@resources.routes.Application.resourceDotFormat("990150876610206441", format="json")</a></p>
 	<p>Einzeltreffer unterstützen zudem verschiedene RDF-Serialisierungen (siehe <a href='#jsonld'>Abschnitt zu JSON-LD</a>).</p>
 
 
@@ -107,22 +107,22 @@ $('input.search-resources').autocomplete({
 	<p>JSON-API-Anfragen liefern <a href="https://json-ld.org/">JSON-LD</a>. Um damit zu experimentieren können sie das JSON-LD oder URLs im <a href="https://json-ld.org/playground/">JSON-LD Playground</a> einfügen.</p>
 	@desc("JSON-LD Context", resources.routes.Application.context)
 	<dt>RDF-Konvertierung</dt><dd>Mit einem JSON-LD-Prozessor können sie API-Antworten in andere RDF-Serialisierungen umwandeln, z.B. in N-Quads mit <a href="https://github.com/digitalbazaar/jsonld-cli">jsonld-cli</a>:</dd>
-	<p><code>jsonld format --nquads https://lobid.org@resources.routes.Application.resource("HT018472857")</code></p>
+	<p><code>jsonld format --nquads https://lobid.org@resources.routes.Application.resource("990150876610206441")</code></p>
 	<p>Einzeltreffer können per <code>Accept</code> Header auch direkt als RDF/XML, Turtle oder N-Triples angefordert werden:</p>
-	<p><code>curl --header "Accept: application/rdf+xml" https://lobid.org@resources.routes.Application.resource("HT018472857")</code></p>
-	<p><code>curl --header "Accept: text/turtle" https://lobid.org@resources.routes.Application.resource("HT018472857")</code></p>
-	<p><code>curl --header "Accept: application/n-triples" https://lobid.org@resources.routes.Application.resource("HT018472857")</code></p>
+	<p><code>curl --header "Accept: application/rdf+xml" https://lobid.org@resources.routes.Application.resource("990150876610206441")</code></p>
+	<p><code>curl --header "Accept: text/turtle" https://lobid.org@resources.routes.Application.resource("990150876610206441")</code></p>
+	<p><code>curl --header "Accept: application/n-triples" https://lobid.org@resources.routes.Application.resource("990150876610206441")</code></p>
 	<p>Dabei werden wie <a href='#content_types'>im Abschnitt zu Inhaltstypen beschrieben</a> neben dem Format-Parameter auch URLs mit Formatendungen unterstützt:
-	<p><a href='@resources.routes.Application.resourceDotFormat("HT018472857", format="rdf")'>@resources.routes.Application.resourceDotFormat("HT018472857", format="rdf")</a></p>
-	<p><a href='@resources.routes.Application.resourceDotFormat("HT018472857", format="ttl")'>@resources.routes.Application.resourceDotFormat("HT018472857", format="ttl")</a></p>
-	<p><a href='@resources.routes.Application.resourceDotFormat("HT018472857", format="nt")'>@resources.routes.Application.resourceDotFormat("HT018472857", format="nt")</a></p>
+	<p><a href='@resources.routes.Application.resourceDotFormat("990150876610206441", format="rdf")'>@resources.routes.Application.resourceDotFormat("990150876610206441", format="rdf")</a></p>
+	<p><a href='@resources.routes.Application.resourceDotFormat("990150876610206441", format="ttl")'>@resources.routes.Application.resourceDotFormat("990150876610206441", format="ttl")</a></p>
+	<p><a href='@resources.routes.Application.resourceDotFormat("990150876610206441", format="nt")'>@resources.routes.Application.resourceDotFormat("990150876610206441", format="nt")</a></p>
 	<dt>Beispieldokumente</dt><dd>Diese Beispieldokumente wurden mit <a href="https://hypothes.is/">hypothes.is</a> annotiert, um den Einstieg in die Datenlage zu erleichtern. Bei Klick auf einen gelbmarkierten JSON-Key geht ein Popup-Fenster auf mit Informationen zu Name, Beschreibung, Abdeckung sowie Anwendungsbeispielen des Feldes.</dd>
 	<dd>
 		<p>Springe zu: <a href="#buch">Buch</a>, <a href="#aufsatz">Aufsatz</a>, <a href="#periodikum">Periodikum</a>, <a href="#serienband">Serienband</a>
-		@sample_doc("HT019025795", "Buch")
-		@sample_doc("HT013577568", "Aufsatz")
-		@sample_doc("HT013304490", "Periodikum")
-		@sample_doc("HT002888677", "Serienband")
+		@sample_doc("990212549810206441", "Buch")
+		@sample_doc("990119006850206441", "Aufsatz")
+		@sample_doc("990116291950206441", "Periodikum")
+		@sample_doc("990091680360206441", "Serienband")
 	</dd>
 	</dl>
 }

